plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraTransform.kt

```diff
@@ -26,11 +26,9 @@ internal object CameraTransform {
 
   fun ScreenCoordinate.offset(arg: ScreenCoordinate) = ScreenCoordinate(x - arg.x, y - arg.y)
 
-  fun getMapCenter(edgeInsets: EdgeInsets?, mapSize: Size): ScreenCoordinate {
-    val centerX = (mapSize.width - (edgeInsets?.left ?: 0.0) - (edgeInsets?.right ?: 0.0)) / 2.0 +
-      (edgeInsets?.left ?: 0.0)
-    val centerY = (mapSize.height - (edgeInsets?.top ?: 0.0) - (edgeInsets?.bottom ?: 0.0)) / 2.0 +
-      (edgeInsets?.top ?: 0.0)
+  fun getMapCenter(edgeInsets: EdgeInsets, mapSize: Size): ScreenCoordinate {
```

Comment:
why do we change the signature? seems not related to the recent changes? can we make it 
```
fun getMapCenter(edgeInsets: EdgeInsets? = null, mapSize: Size): ScreenCoordinate {
```

Reply:
it's related to gl-native - now edge insets are always non-null